client.go

```diff
@@ -219,7 +219,7 @@ func (s *Client) Run(ctx context.Context,
 		info.Alias, hex.EncodeToString(info.IdentityPubkey[:]),
 	)
 
-	// Setup main context used for cancelation.
+	// Setup main context used for cancellation.
```

Comment:
This is a hot debate. @cfromknecht, @carlaKC, ... ?

Reply:
The great debate rages on 🔥